packages/react-router-dom/server.tsx

```diff
@@ -119,22 +119,18 @@ export function StaticRouterProvider({
 
   return (
     <>
-      <DataStaticRouterContext.Provider value={context}>
```

Comment:
This outer `<DataStaticRouterContext>` is removed in favor of a `staticContext` field on `dataRouterContext`.  Easiest to see with whitespace hidden 😉